### PR TITLE
admin: make user-info table selectable

### PR DESCRIPTION
**Pull request by johnkpark**
Screen Shot 2022-05-05 at 9 48 12 AM

`admin-table-head` class sets `user-select` to `none`, which makes text inside the table un-selectable. This is inconvenient when you need to copy some properties while debugging.

### CODE REVIEW

**Comment by minrk** [comment]
@naatebarber @NarekA review?

**Comment by naatebarber** [comment]
LGTM :+1:

File: jsx/src/components/ServerDashboard/ServerDashboard.jsx [hunk 1]
```diff
@@ -1,5 +1,4 @@
 import React, { useState } from "react";
-import regeneratorRuntime from "regenerator-runtime";
```
Comment (NarekA) [inline jsx/src/components/ServerDashboard/ServerDashboard.jsx:2]: Are you sure this is safe to remove? I remember I had to add this import to fix an error, but I forget what it was.

File: jsx/src/components/ServerDashboard/ServerDashboard.jsx [hunk 2]
```diff
@@ -200,6 +199,25 @@ const ServerDashboard = (props) => {
...
+        keyStyle={{
+          padding: "4px",
+        }}
+        valueStyle={{
+          padding: "4px",
+        }}
+        data={data}
+      />
+    );
+  };
```
Comment (NarekA) [inline jsx/src/components/ServerDashboard/ServerDashboard.jsx:219]: I see you have continued the effort to un-nest this module over time.

**Review by NarekA — APPROVED** [review]
Good job tracking down the cause!

File: jsx/src/components/ServerDashboard/ServerDashboard.jsx (hunk 1 above)
Comment (johnkpark) [reply jsx/src/components/ServerDashboard/ServerDashboard.jsx:2]: The linter removed this for me. I think it's safe to remove as long as it's not used within this file.

Comment (naatebarber) [reply jsx/src/components/ServerDashboard/ServerDashboard.jsx:2]: Good catch, There's a chance it'll throw an error if that file uses async await syntax. I'd leave it in

Comment (naatebarber) [reply jsx/src/components/ServerDashboard/ServerDashboard.jsx:2]: Meant to whitelist that import in the .eslintrc

Comment (johnkpark) [reply jsx/src/components/ServerDashboard/ServerDashboard.jsx]: Interesting. I've added this back in.

**Comment by johnkpark** [comment]
@minrk can we merge this?

**Comment by minrk** [comment]
Thanks all!

**Comment by yuvipanda** [comment]
@MeeseeksDev backport 2.x

**Comment by consideRatio** [comment]
This PR is part of the 2.3.1 release

**Pull request merged**